cmd/jobbuzz-scraper/main.go

```diff
@@ -4,13 +4,22 @@ import (
 	"encoding/json"
 	"fmt"
 
+	"github.com/b-open/jobbuzz/internal/config"
 	"github.com/b-open/jobbuzz/pkg/scraper"
 )
 
 func main() {
+	db := config.GetDb()
+
+	config.MigrateDb(db)
```

Comment:
I think it might be better to have the migrator run on a separate cmd.

Reason being normally orm database migration it is normally recommended not to run auto migration on every startup.

It is especially dangerous here since we now have 2 different program that is very likely to both try to run the migration at the same time which could cause some weird issues.

According to this repo below, gorm's migration might cause issue if you're running multiple instances of the migrator.

https://github.com/go-gormigrate/gormigrate